xarray/core/common.py

```diff
@@ -343,17 +343,21 @@ def groupby(self, group, squeeze=True):
             group = self[group]
         return self.groupby_cls(self, group, squeeze=squeeze)
 
-    def rolling(self, min_periods=None, center=False, **kwarg):
-        """Returns a Rolling object for performing moving window operations.
+    def rolling(self, min_periods=None, center=False, **windows):
+        """
+        Moving window object.
 
         Parameters
         ----------
+        obj : Dataset or DataArray
+            Object to window.
         min_periods : int, default None
             Minimum number of observations in window required to have a value
-            (otherwise result is NA).
+            (otherwise result is NA). The default, None, is equivalent to
+            setting min_periods equal to the size of the window.
```

Comment:
I think bottleneck and pandas differ on how they handle the `min_periods` argument.  

Bottleneck:

>min_count: {int, None}, optional :
>
>    If the number of non-NaN values in a window is less than min_count, then a value of NaN is assigned to the window. By default min_count is None, which is equivalent to setting min_count equal to window.

Pandas doesn't say in its doc string: 

>min_periods : int, default None
>
>    Minimum number of observations in window required to have a value (otherwise result is NA).

So, comparing their behavior, we see they both set `min_periods` to the size of the window.

```Python
In [1]: import pandas as pd
In [2]: s = pd.Series(range(8))    
In [3]: pd.rolling_mean(s, 3)
Out[3]: 
0   NaN
1   NaN
2     1
3     2
4     3
5     4
6     5
7     6
dtype: float64
In [4]: import bottleneck as bn
In [6]: bn.move_mean(s, 3)
Out[6]: array([ nan,  nan,   1.,   2.,   3.,   4.,   5.,   6.])